### PR TITLE
Fix invalid test properties and css value

### PR DESCRIPTION
**Pull request by minevala**
## Description
- Fix invalid CSS value in Tabs.module.scss
- Add type-check phase to lint npm-script (checks also test files)
- Fix test's invalid types and test snapshots

## Closes
- https://helsinkisolutionoffice.atlassian.net/browse/HDS-797

## Motivation and Context
- As part of release 1.0 preparations, we try to clean up component code.

## How Has This Been Tested?
- Locally on dev's machine

### CODE REVIEW

**Review by laurakarhu — APPROVED** [review]
No more "foo", yay!

File: packages/react/src/components/tabs/Tabs.module.scss [hunk 1]
```diff
@@ -143,4 +143,4 @@
   }
 }
 
-.tabPanel {}
\ No newline at end of file
+.tabPanel {}
```
Comment (VilleMiekkoja) [inline packages/react/src/components/tabs/Tabs.module.scss:146]: Empty style declaration?

**Review by VilleMiekkoja — APPROVED** [review]
👍 Nice

File: packages/react/src/components/tabs/Tabs.module.scss (hunk 1 above)
Comment (minevala) [reply packages/react/src/components/tabs/Tabs.module.scss:146]: Good catch :thi I will remove this.

**Review by VilleMiekkoja — APPROVED** [review]
Good job 🏅

**Pull request merged**